alf/utils/external_configurables.py

```diff
@@ -41,10 +41,6 @@
 gin.external_configurable(torch.relu, 'torch.relu')
 gin.external_configurable(torch.nn.functional.elu, 'torch.nn.functional.elu')
 
-gin.external_configurable(alf.TensorSpec, 'alf.TensorSpec')
-
-gin.external_configurable(alf.BoundedTensorSpec, 'alf.BoundedTensorSpec')
-
```

Comment:
Why these lines are removed?
Test cases are broken when these are removed. For example, in ```diayn_pendulum.gin```, ```goal/alf.TensorSpec.shape=(%skill_feature_size,)``` is used.
```ppo_rnd_mrevenge.gin``` and other cases may have a similar issue.

Reply:
Just use TensorSpec without prefixing 'alf.'

Reply:
It seems the ambiguity error are due to the following ```gin.configurable ``` introduced in #439:
https://github.com/HorizonRobotics/alf/blob/f4ae6a4ea9db05739ff63f0ef7923409cd4d5565/alf/tensor_specs.py#L30-L31
https://github.com/HorizonRobotics/alf/blob/f4ae6a4ea9db05739ff63f0ef7923409cd4d5565/alf/tensor_specs.py#L168-L169
Why are they necessary? Can we simply remove them to resolve the ambiguity error you mentioned? 
In addition, I feel the original form (```alf.TensorSpec```) might be more clear in indicating it is from alf and also configuring them in a centralized place (```external_configurables.py```) seems to be a good idea.

Reply:
I think external_configurables.py suggests that all those configurations inside that file have to be done in that way because they are "external" (third-party code). We have our own implementation of TensorSpec, and it makes little sense to treat them as external.